public/Player.js

```diff
@@ -32,22 +36,98 @@ export default class Player {
     this.domElement = htmlPlayersContainer.querySelector(`#${this.name}`);
   }
 
+  initialBet() {}
+
+  playerBetValurHtml() {
+    return `
+    Apuesta:${this.betValue}
+    `;
+  }
+
+  renderBetValue() {}
+
+  sumOfCards() {
+    let sumOfHand = 0;
+
+    const arrOfCardsWithFixedValues = this.hand.filter(
+      (card) => !card.alternateValue
+    );
+
+    const arrOfCardsWithAlternateValues = this.hand.filter(
+      (card) => card.alternateValue
+    );
+
+    let amountOfAces = arrOfCardsWithAlternateValues.length;
+    let alternateValuesUsed = 0;
+    const amountReducedIfAceChangesTo1 = 10;
+
+    for (let card of arrOfCardsWithFixedValues) {
+      if (card.faceDirection === "down") break;
+      sumOfHand += card.value;
+    }
+
+    if (amountOfAces === 0) {
+      this.handValue = sumOfHand;
+      return;
+    }
+
+    for (let card of arrOfCardsWithAlternateValues) {
+      if (card.faceDirection === "down") break;
+      if (sumOfHand + card.value > 21) {
+        sumOfHand = sumOfHand + card.alternateValue;
```

Comment:
This can be simplified to:
```suggestion
        sumOfHand += card.alternateValue;
```